### PR TITLE
Run helm install test in create release without prow workflow

### PR DESCRIPTION
**Pull request by MarekMichali**
**Description**

This PR prepares the new release workflow that uses the GitHub Actions image-build to use helm install. While we are still using the old release workflow with prow, the `run-keb-chart-install-tests-reusable.yaml` file has been temporarily duplicated to not break the old workflow.

Changes proposed in this pull request:

- Add postgres config for tests
- Add temporary `run-keb-chart-install-tests-reusable.yaml` workflow
- Run helm install in release without prow

**Related issue(s)**

See #831

### CODE REVIEW

File: .github/workflows/create-release-without-prow.yaml [hunk 1]
```diff
@@ -133,10 +141,10 @@ jobs:
       build-args: BIN=subaccount-sync
       tags: ${{ inputs.name }}
 
-  run-keb-chart-tests:
+  run-keb-chart-install-tests:
     name: Validate KEB chart 
     needs: [build-keb-image, build-archiver-image, build-environments-cleanup-image, build-deprovision-retrigger-image, build-expirator-image, build-runtime-reconciler-image, build-subaccount-cleanup-image, build-subaccount-sync-image]
-    uses: "./.github/workflows/run-keb-chart-tests-reusable.yaml"
+    uses: "./.github/workflows/run-keb-chart-install-tests-reusable.yaml"
```
Comment (ralikio) [inline .github/workflows/create-release-without-prow.yaml:147]: Is `run-keb-chart-tests-reusable.yaml` still used somewhere?

Comment (MarekMichali) [reply .github/workflows/create-release-without-prow.yaml:147]: In the current release workflow where we are using prow.

Comment (MarekMichali) [reply .github/workflows/create-release-without-prow.yaml:147]: And in tests on PR's.

File: .github/workflows/run-keb-chart-install-tests-reusable.yaml [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+      with:
+        files: |
+          .github/**
+          resources/**
+          scripts/**
+          Makefile
+          **/*.sh
+
+    - name: Prepare chart for next KEB version
+      if: inputs.release == 'true'
```
Comment (ralikio) [inline .github/workflows/run-keb-chart-install-tests-reusable.yaml:39]: do we have to use comparison in this iff statement?

Comment (MarekMichali) [reply .github/workflows/run-keb-chart-install-tests-reusable.yaml:39]: Yes, this is a string.

**Review by ralikio — APPROVED** [review]

File: .github/workflows/create-release-without-prow.yaml (hunk 1 above)
Comment (ralikio) [reply .github/workflows/create-release-without-prow.yaml:147]: This is just a temporary for created for test convenience.

File: .github/workflows/run-keb-chart-install-tests-reusable.yaml (hunk 2 above)
Comment (ralikio) [reply .github/workflows/run-keb-chart-install-tests-reusable.yaml:39]: We have agreed to check if boolean works later on.

**Pull request merged**